composer/datasets/c4.py

```diff
@@ -40,7 +40,7 @@ def _split_dict_fn(batch: Batch, n_microbatches: int) -> List[Batch]:
 class C4DatasetHparams(DatasetHparams):
     """Builds a DataSpec for the C4 (Colossal Cleaned CommonCrawl) dataset.
 
-    Parameters:
+    Args:
```

Comment:
The reference to `composer.core.DataSpec` in the "Returns" section of the docstring isn't resolving.

Reply:
The "Returns" section should also probably be in a docstring for `initialize_object`